src/main/java/com/google/ehub/servlets/ProfileServlet.java

```diff
@@ -17,46 +21,58 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** 
-* Handles adding the form submission to the datastore in 
-* POST and returns json of the user's profile information
-* to client.
-*/
+/**
+ * Handles adding the form submission to the datastore in
+ * POST and returns json of the user's profile information
+ * to client.
+ */
 @WebServlet("/profile-data")
 public class ProfileServlet extends HttpServlet {
-  
   private static final String NAME_PROPERTY_KEY = "name";
   private static final String EMAIL_PROPERTY_KEY = "email";
   private static final String USERNAME_PROPERTY_KEY = "username";
   private static final String BIO_PROPERTY_KEY = "bio";
-  ProfileDatastore profileData = new ProfileDatastore();
+  private static final String EDIT_PROPERTY_KEY = "edit";
+  
+  private static final UserService userService = UserServiceFactory.getUserService();
+  private static final ProfileDatastore profileData = new ProfileDatastore();
```

Comment:
```suggestion
  private final UserService userService = UserServiceFactory.getUserService();
  private final ProfileDatastore profileData = new ProfileDatastore();
```

These can be non-static so their lifecycle is tied to the lifecycle of a ProfileServlet instance.